File: ANEFreeInIty_AI/AIBackEnd/Data/Entity/Project.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace AIBackEnd.Data.Entity
{
    [Table("project")]
    public class Project
    {
        [Key]
        [Required(ErrorMessage = "Project ID is required")]
        [Column("id")]
        public int Id { get; set; }

        [Required(ErrorMessage = "Name is required")]
        [Column("p_name")]
        public string Name { get; set; }

        [Required(ErrorMessage = "Description is required")]
        [Column("p_description")]
        public string Description { get; set; }

        virtual public IEnumerable<ProjectIteamMapper> ProjectMapper { get; set; }
    }
}

File: FrontEnd/AIFrontEND/src/app/Model/Project.ts
import { ProjectIteamMapper } from "./ProjectIteamMapper";

export class Project {
    id: number | undefined | null;
    name: string | undefined | null = "no name";
    description: string | undefined | null = "none";
    projectMapper: ProjectIteamMapper[];
}

export interface IProject {
    id: number;
    name: string;
    description: string;
    projectMapper: ProjectIteamMapper[];
}